table/update_schema.go

```diff
@@ -191,14 +189,12 @@ func (us *UpdateSchema) Apply() *iceberg.Schema {
 	return us.applyChanges()
 }
 
-// ══════════════════════════════INTERNAL══════════════════════════════════════║
-
 func (su *UpdateSchema) isAdded(name string) bool {
 	_, ok := su.addedNameToID[name]
 	return ok
 }
 
-func (us *UpdateSchema) internalAddColumn(parent, name string, new_id int, required bool, dataType iceberg.Type, doc string, initialDefaultValue any) *UpdateSchema {
+func (us *UpdateSchema) InternalAddColumn(parent, name string, new_id int, required bool, dataType iceberg.Type, doc string, initialDefaultValue any) *UpdateSchema {
```

Comment:
it's just `AddColumn` now, internal nature of it is artificial, no need for overcomplicated things :D 
keep it simple,